feconf.py

```diff
@@ -420,6 +420,15 @@ def get_empty_ratings():
 # with the Mailgun domain name (ending with mailgun.org).
 MAILGUN_DOMAIN_NAME = None
 
+# Audience ID of the mailing list for Oppia in Mailchimp.
+MAILCHIMP_AUDIENCE_ID = None
+
```

Comment:
Maybe drop the blank lines between these three constants, it makes sense to group them.

Reply:
Done